downloads/index.md

```diff
@@ -1,39 +1,19 @@
 ---
-layout: downloads
+layout: downloads-index-layout
 title: "Download"
 subtitle: ""
-description: Download Iridium Browser for Windows, macOS, Ubuntu, Debian, Mint, OpenSUSE, Fedora and Red Hat Enterprise Linux
+description: Download Iridium Browser for Windows, macOS, Ubuntu, Debian, Mint, OpenSUSE, Fedora, Red Hat Enterprise Linux 7 CentOS
```

Comment:
inconsistent specification of platforms. In other places, CentOS is listed as a separate entity rather than as a flavor of RHEL.

Reply:
type-o